components/standings/standings_storage.lua

```diff
@@ -0,0 +1,36 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Standings/Storage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local StandingsStorage = {}
+
+function StandingsStorage.run(index, data)
+	mw.ext.LiquipediaDB.lpdb_standing(
+		'standing' .. index,
+		{
+			title = data.title,
+			tournament = data.tournament,
+			participant = data.participant,
+			participantdisplay = data.participantdisplay,
+			participantflag = data.participantflag,
+			icon = data.icon,
+			placement = data.placement or data.rank,
+			definitestatus = data.definitestatus or data.bg,
+			currentstatus = data.currentstatus or data.pbg,
+			change = data.change,
+			scoreboard = mw.ext.LiquipediaDB.lpdb_create_json({
```

Comment:
do we maybe want to have the following added to scoreboard?
```lua
				finalTiebreak = data.finalTiebreak, --in case normal tie breaker rules did not solve the ties and we manually broke them
				customPoints = data.customPoints, --in case we added manual points that were not queried by matches
				diff = data.diff, --in case the diff is to be displayed too (could be set as something different from pure games won - games loss, depending on the wikis
```

Reply:
Added diff, don't know what those others are and why they would be useful in the app

Reply:
alright^^